packages/hadron-build/lib/mac-notary-service.js

```diff
@@ -0,0 +1,95 @@
+const download = require('download');
```

Comment:
```suggestion
'use strict';
const download = require('download');
```

(this is not an ES/TS module so not strict by default)